### ISSUE TITLE
Coursera - Machine Learning(2)

### ISSUE THREAD

**Issue by BanSugyeong** [BanSugyeong/Machine-Learning-Study#2]
### Advanced Learning Algorithms

**Comment by BanSugyeong** [comment]
### Contents
- Feedback
  - implementation of MSE
  - implementation of RMSE
  - overfitting solutions
  - L1, L2 regularization
- Neural Networks
  - Layers
  - Forward Propagation
- Perceptron
  - single-layer perceptron
  - multi-layer perceptron
- Tensorflow

**Comment by BanSugyeong** [comment]
### Contents
- Activation Functions
  - Sigmoid function
  - ReLU function
  - ELU function
  - softmax function
- Multiclass Classification
  - Loss function of Multiclass Classification
- Back Propagation

[240126_MachineLearning_BanSugyeong.pdf](https://github.com/BanSugyeong/Machine-Learning-Study/files/14382112/240126_MachineLearning_BanSugyeong.pdf)

**Comment by BanSugyeong** [comment]
## Contents
- Feedback
  - Activation Functions
    - Binary step function
    - Linear activation function
    - Non-linear activation function
- Evaluating a model
- Bias and Variance
- Precision and Recall
  - Trading off percision and recall
- Decision tree
  - Pruning and Ensemble Methods
  - Impurity
- Random forest

[240202_MachineLearning_BanSugyeong.pdf](https://github.com/BanSugyeong/Machine-Learning-Study/files/14403506/240202_MachineLearning_BanSugyeong.pdf)
